docs/guides/qiskit-code-assistant-local.mdx

```diff
@@ -24,18 +24,80 @@ The Open Source Qiskit Code Assistant models are available in <DefinitionTooltip
+
+1. [qiskit/granite-3.3-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit-GGUF) - Released June 2025
+2. [qiskit/granite-3.2-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit-GGUF) - Released June 2025
+3. [qiskit/qwen2.5-coder-14b-qiskit-GGUF](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit-GGUF) - Released June 2025
+
+#### Qiskit Versions Used for Training
+
+<table
+  style={{
+    display: 'inline-table',
+    borderCollapse: 'separate',
+    borderSpacing: 0,
+    fontFamily: "Inter, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif",
+    boxShadow: '0 6px 18px rgba(12, 20, 29, 0.06)',
+    borderRadius: '12px',
+    overflow: 'hidden',
+    tableLayout: 'auto',
+    boxSizing: 'border-box',
+    margin: '16px 0'
+  }}
+>
+  <thead>
+    <tr>
+      <th style={{textAlign: 'left', padding: '10px 14px', background: '#e8f0f8', color: '#475569', fontWeight: 600, fontSize: '13px', borderBottom: '1px solid rgba(15,23,42,0.08)'}}>
+        Model
+      </th>
+      <th style={{textAlign: 'center', padding: '10px 14px', background: '#e8f0f8', color: '#475569', fontWeight: 600, fontSize: '13px', borderBottom: '1px solid rgba(15,23,42,0.08)'}}>
+        Trained on Qiskit Version
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr style={{background: '#ffffff'}}>
+      <td style={{padding: '10px 14px', fontWeight: 700, color: '#0f172a', fontSize: '14px'}}>mistral-small-3.2-24b-qiskit</td>
+      <td style={{padding: '10px 14px', fontWeight: 700, textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.1</td>
+    </tr>
+    <tr style={{background: '#fbfdff'}}>
+      <td style={{padding: '10px 14px', color: '#0f172a', fontSize: '14px'}}>granite-3.3-8b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
+    <tr style={{background: '#ffffff'}}>
+      <td style={{padding: '10px 14px', color: '#0f172a', fontSize: '14px'}}>granite-3.2-8b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
+    <tr style={{background: '#fbfdff'}}>
+      <td style={{padding: '10px 14px', color: '#07102a', fontSize: '14px'}}>qwen2.5-coder-14b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Deprecated Models
+
+These models are no longer actively maintained but remain available:
+
+1. [qiskit/granite-8b-qiskit-rc-0.10](https://huggingface.co/Qiskit/granite-8b-qiskit-rc-0.10) - Released February 2025 (deprecated)
+2. [qiskit/granite-8b-qiskit](https://huggingface.co/Qiskit/granite-8b-qiskit) - Released November 2024 (deprecated)
```

Comment:
It would make sense to document here up to which Qiskit version they were updated to.